docs/source/conf.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# torchtext documentation build configuration file, created by
+# sphinx-quickstart on Thu Nov 16 01:05:05 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+import torch
+import torchtext
+import sphinx_rtd_theme
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
+    'sphinx.ext.doctest',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.todo',
+    'sphinx.ext.coverage',
+    'sphinx.ext.mathjax',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.viewcode',
+]
+
+napoleon_use_ivar = True
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'torchtext'
+copyright = '2017, Torch Contributors'
+author = 'Torch Contributors'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = '0.2.0'
```

Comment:
perhaps it's better to just import the package and use `package.__version__`? e.g. https://github.com/pytorch/pytorch/blob/master/docs/source/conf.py#L78

Reply:
In order to do this, I made some changes on setup.py
https://github.com/keon/text/blob/5003ffb56017768c1dbafae81587321c5652729b/setup.py

and saved version inside `__version__` in `torchtext/__init__.py`

Reply:
whoops, didn't realize that `__version__` was not yet set --- thanks for handling that. i would have used something like https://github.com/kennethreitz/setup.py/blob/master/setup.py#L38 and https://github.com/kennethreitz/setup.py/blob/master/mypackage/__version__.py, but this looks like it should work too.

Reply:
how about this? https://github.com/keon/text/commit/7c68bb1b4268b498fe2add77cdc6be5d154c1729